contrib/auto-render/auto-render.js

```diff
@@ -62,7 +64,7 @@ const renderElem = function(elem, delimiters, ignoredTags) {
                 childNode.nodeName.toLowerCase()) === -1;
 
             if (shouldRender) {
-                renderElem(childNode, delimiters, ignoredTags);
+                renderElem(childNode, delimiters, ignoredTags, renderOptions);
```

Comment:
Same here: we can just pass in the options argument (which would probably be better named `options` or `optionsCopy`?) instead of explicitly calling out `delimeters` and `ignoredTags`.